app/networkConfig.ts

```diff
@@ -1,15 +1,21 @@
 import { getFullnodeUrl } from "@mysten/sui/client";
 import { createNetworkConfig } from "@mysten/dapp-kit";
+import swapTestNetConfig from "./config/sui/contracts-testnet.json";
+import swapMainNetConfig from "./config/sui/contracts-mainnet.json";
 
 const { networkConfig, useNetworkVariable, useNetworkVariables } = createNetworkConfig({
-	devnet: {
-		url: getFullnodeUrl("devnet"),
-	},
 	testnet: {
 		url: getFullnodeUrl("testnet"),
+		variables: {
+			...swapTestNetConfig,
+		},
 	},
 	mainnet: {
 		url: getFullnodeUrl("mainnet"),
+		variables: {
+			hello: "hello",
```

Comment:
What is variable hello here?

Reply:
it was just for testing. thanks for pointing out.